samples/migrations/pg_to_alloy.md

```diff
@@ -0,0 +1,136 @@
+
+## How to migrate
+
+1. Create an AlloyDB engine.
+
+    ```python
+    from langchain_google_alloydb_pg import AlloyDBEngine
+
+    # Replace these variable values
+    engine = await AlloyDBEngine.afrom_instance(
+        project_id="my-project-id",
+        instance="my-instance-name",
+        region="us-central1",
+        cluster="my-primary",
+        database="test_db",
+        user="user",
+        password="password",
+    )
+    ```
+
+    > **_NOTE:_** All async methods have corresponding Sync Methods.
+
+2. Create a new table to migrate existing data
+
+    ```python
+    # Vertex AI embeddings uses a vector size of 768. Change this according to your embeddings service.
+    VECTOR_SIZE = 768
+
+    await engine.ainit_vectorstore_table(
+        table_name="destination_table",
+        vector_size=VECTOR_SIZE,
+    )
+    ```
+
+    You can customise the table by using custom metadata or id columns as follows:
+
+    ```python
+    metadata_columns = [
+        Column(f"col_0_{collection_name}", "VARCHAR"),
+        Column(f"col_1_{collection_name}", "VARCHAR"),
+    ]
+    await engine.ainit_vectorstore_table(
+        table_name=collection_name,
+        vector_size=VECTOR_SIZE,
+        metadata_columns=metadata_columns,
+        id_column=Column("langchain_id", "VARCHAR"),
+    )
+    ```
+
+    You can refer to the [docs](https://cloud.google.com/python/docs/reference/langchain-google-alloydb-pg/latest/langchain_google_alloydb_pg.alloydb_vectorstore.AlloyDBVectorStore#langchain_google_alloydb_pg_alloydb_vectorstore_AlloyDBVectorStore_create) for any vector store customisations.
+
+3. Create a vector store object to interact with the new data.
+
+    ```python
+    from langchain_google_alloydb_pg import AlloyDBVectorStore
+    from langchain_core.embeddings import FakeEmbeddings
+
+    vector_store = await AlloyDBVectorStore.create(
+        engine,
+        embedding_service=FakeEmbeddings(size=VECTOR_SIZE),
+        table_name="destination_table",
+    )
+    ```
+
+    If you have any metadata columns, add them to the vector store as follows:
+
+    ```python
+    from langchain_google_alloydb_pg import AlloyDBVectorStore
+    from langchain_core.embeddings import FakeEmbeddings
+
+    vector_store = await AlloyDBVectorStore.create(
+        engine,
+        embedding_service=FakeEmbeddings(size=VECTOR_SIZE),
+        table_name="destination_table",
+        metadata_columns=[col.name for col in metadata_columns],
+    )
+    ```
+
+    > **_NOTE:_** The Fake Embeddings embedding service is only used to initialise a vector store object, not to generate any embeddings. The embeddings are directly copied from the PGVector database.
+
+4. Migrate data to the new table
+
+    ```python
+    from langchain_google_alloydb_pg.utils.pgvector_migrator import amigrate_pgvector_collection
+
+    await amigrate_pgvector_collection(
```

Comment:
It would also be helpful to add some details on what happens if this doesn't succeed. For example, the docs won't be deleted unless it's all moved.